pages/api/onboardUser.ts

```diff
@@ -0,0 +1,52 @@
+import { NextApiRequest, NextApiResponse } from "next";
+const SibApiV3Sdk = require("@getbrevo/brevo");
+
+export default async function onboardUser(
```

Comment:
This feels like something that should be done server side as if anything goes wrong with this request the two lists are going to end up out of sync. 

If you want to go with this approach this endpoint also needs to check that the email is authenticated with web3auth otherwise anyone can use this endpoint to spam our mailing list with fake signups